doc/release-notes.md

```diff
@@ -1,33 +1,45 @@
 (note: this is a temporary file, to be added-to by anybody, and moved to
 release-notes at release time)
 
-Bitcoin Core version 0.14.0 is now available from:
+Dogecoin Core version 0.14.0 is now available from:
 
-  <https://bitcoin.org/bin/bitcoin-core-0.14.0/>
+  <https://dogecoin.com/bin/dogecoin-core-0.14.0/>
```

Comment:
That URL is probably wrong. Should point to GitHub maybe?

Reply:
I've rewritten it to point to nowhere that works in Github, but mostly it's a placeholder until we get to the release itself.